torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -31,26 +31,27 @@ def _get_random_item(self, dct: Dict[K, V]) -> Tuple[K, V]:
         key = keys[int(torch.randint(len(keys), ()))]
         return key, dct[key]
 
-    def _extract_image(
+    def _extract_image_or_video(
         self,
         sample: Any,
         unsupported_types: Tuple[Type, ...] = (features.BoundingBox, features.Mask),
-    ) -> Tuple[int, features.ImageType]:
+    ) -> Tuple[int, features.ImageOrVideoType]:
         sample_flat, _ = tree_flatten(sample)
-        images = []
+        image_likes = []
```

Comment:
Nit?
```suggestion
        inputs = []
```

Reply:
I renamed to `image_or_videos` to be on par with the `image_or_video` variable in the actual transforms. I didn't like `inputs` here, because that would imply `input = self._extra_image_or_video(...)` and that clashes with the name. Plus, `input` is already used as variable in the actual transformation.